manifests/charts/gateway/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
If the option to not create the role bindings is set - maybe we can also use default SA instead of creating a dedicated SA.

Reply:
I will look into common practice. I think its still useful for things like WI, AuthorizationPolicies, etc.